qwandaq/src/main/java/life/genny/qwandaq/exception/runtime/config/MissingEnvironmentVariableException.java

```diff
@@ -0,0 +1,12 @@
+package life.genny.qwandaq.exception.runtime.config;
+
+import life.genny.qwandaq.exception.GennyRuntimeException;
+
+public class MissingEnvironmentVariableException extends GennyRuntimeException {
+    
+    private static final String MESSAGE = "Missing environment variable: ";
+
+    public MissingEnvironmentVariableException(String environemtVariable) {
+        super(MESSAGE.concat(environemtVariable));
+    }
+}
```

Comment:
This will be the start of upgrading GennySettings.java when we have time, it would also be a good small task for Tom and Thabo getting them used to the system. currently a single error line is used and the system proceeds and runs into a NullPointer or something if there is a missing environment variable